<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-0BGE448" failures="3" tests="5" name="Test" time="14.981" errors="0" timestamp="01 Feb 2022 06:53:47 GMT">
  <testcase classname="TestCases.AddPayeeDatadriven" name="login" time="3.241"/>
  <testcase classname="TestCases.AddPayeeDatadriven" name="addPayee" time="1.223"/>
  <testcase classname="TestCases.AddPayeeDatadriven" name="addPayee" time="0.899">
    <failure type="java.lang.AssertionError" message="expected [The new payee Payee1 was successfully created.] but found [The new payee Payee2 was successfully created.]">
      <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee2 was successfully created.]
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:82)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addPayee -->
  <testcase classname="TestCases.AddPayeeDatadriven" name="addPayee" time="0.921">
    <failure type="java.lang.AssertionError" message="expected [The new payee Payee1 was successfully created.] but found [The new payee Payee3 was successfully created.]">
      <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee3 was successfully created.]
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:82)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addPayee -->
  <testcase classname="TestCases.AddPayeeDatadriven" name="addPayee" time="0.876">
    <failure type="java.lang.AssertionError" message="expected [The new payee Payee1 was successfully created.] but found [The new payee Payee4 was successfully created.]">
      <![CDATA[java.lang.AssertionError: expected [The new payee Payee1 was successfully created.] but found [The new payee Payee4 was successfully created.]
	at TestCases.AddPayeeDatadriven.addPayee(AddPayeeDatadriven.java:82)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addPayee -->
</testsuite> <!-- Test -->
